src/evaluator/modules/outputs/builtins/lint-markdown/default.nix

```diff
@@ -1,44 +1,40 @@
-{ __nixpkgs__
-, asBashArray
-, makeDerivation
-, path
+{ __toModuleOutputs__
+, lintMarkdown
+, pathCopy
 , ...
 }:
 { config
 , lib
 , ...
 }:
+let
+  makeOutput = name: { config, targets }: {
+    name = "/lintMarkdown/${name}";
+    value = lintMarkdown {
+      inherit name;
+      config = pathCopy config;
+      targets = builtins.map pathCopy targets;
+    };
+  };
+in
 {
   options = {
-    lintMarkdown = {
-      enable = lib.mkOption {
-        default = false;
-        type = lib.types.bool;
-      };
-      targets = lib.mkOption {
-        default = [ "/" ];
-        type = lib.types.listOf lib.types.str;
-      };
+    lintMarkdown = lib.mkOption {
+      default = { };
+      type = lib.types.attrsOf (lib.types.submodule (_: {
+        options = {
+          config = lib.mkOption {
+            type = lib.types.str;
+            default = "/src/evaluator/modules/outputs/builtins/lint-markdown/config.rb";
```

Comment:
you cannot run pathCopy nor path over this value because "/src/evaluator/modules/outputs/builtins/lint-markdown/config.rb" does not exist in the user project, it exists on makes

Reply:
Do I have to use 

```
        default = ./config.js;
        type = lib.types.path;
```

then?

Reply:
Done.